### PR TITLE
Update docs to reflect the latest CLI command structure

### LINKED ISSUE #14: Update CLI docs
Update the [CLI page](https://docs.vijil.ai/cli/commands.html) when iteration on CLI is done

### PR DESCRIPTION
**Pull request by rajkumaar23**
Closes #14

### CODE REVIEW

File: source/cli/commands.md [hunk 1]
```diff
@@ -1,194 +1,216 @@
...
+| `--token` | String | Token for accessing the provided model. |
 
-### Example
+**Example**
 
 ```console
-foo@bar:~$ vijil run --job-name jobname --model-type octo --model-name mistral-7b-instruct-fp16 --dimension ethics --deployment-type public --generations 1 --token 'your-token'
-Running evaluation for model type: octo, model name: mistral-7b-instruct-fp16
-Successfully Create Evaluation, Check Job Status by ID: 81f05220-27be-4a84-b707-354a71b7359e
+$ vijil start evaluation --model-hub 'octo' --model-name 'mistral-7b-instruct-fp16' --dimension 'ethics' --deployment-type 'public' --generations 1 --token 'your-token'
```
Comment (shubhobm) [inline source/cli/commands.md:94]: I think you can leave out the quotes

Comment (rajkumaar23) [reply source/cli/commands.md:94]: I added quotes for color-coding the difference between the key and the value.

Screenshot 2024-02-21 at 2 41 22 PM

I can remove them if that's not necessary.

Comment (shubhobm) [reply source/cli/commands.md:94]: aha, I see your point. I'm worried about users getting wrong ideas of how to use the commands, which ranks slightly higher than readability in my mind. one middle-ground would be to separate backslash+new line, eg
```console
vijil start evaluation --model-type octo \
   --model-name mistral-7b-instruct-fp16 \
   --dimension Security \
   --deployment-type public \
   --generations 1 \
   --token your-token
```

Comment (rajkumaar23) [reply source/cli/commands.md]: Resolved https://github.com/vijilAI/docs/pull/15/commits/933ccea8330cd2542b744387736a15929a58813c

**Pull request merged**